src/index.js

```diff
@@ -1 +1,77 @@
 import './styles.scss';
+
+let leaderboard = [];
+
+const user = document.getElementById('Inputname');
+const score = document.getElementById('Inputscore');
+const table = document.querySelector('.table_container');
+const form = document.querySelector('.form');
+
+const getScores = async () => {
+  try {
+    const response = await fetch(
+      'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/BenKGame/scores',
+    );
+    const json = await response.json();
+    leaderboard = json.result;
+
+    // Sort the leaderboard based on scores in descending order
+    leaderboard.sort((a, b) => b.score - a.score);
+
+    table.innerHTML = '';
+    leaderboard.forEach((dat) => {
+      //   console.log(table);
+      const row = document.createElement('tr');
+      const name = document.createElement('td');
+      name.className = 'point';
+      name.textContent = `${dat.user}: ${dat.score}`;
+      row.appendChild(name);
+      table.appendChild(row);
+    });
+  } catch (error) {
+    // Handle the error
+  }
+};
+
+const addScore = async (userName, userScore) => {
+  if (!userName || !userScore) {
+    // Display an error message or perform appropriate action for empty inputs
+    return;
+  }
+
+  try {
+    const response = await fetch(
+      'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/BenKGame/scores',
+      {
+        method: 'POST',
+        body: JSON.stringify({
+          user: userName,
+          score: Number(userScore),
+        }),
+        headers: {
+          'Content-Type': 'application/json',
+        },
+      },
+    );
+    const json = await response.json();
+    if (json.response === 'Leaderboard score created.') {
+      getScores();
+    }
+    form.reset();
+    // user.value = '';
+    // score.value = '';
+  } catch (error) {
```

Comment:
- [ ] Great job on hitting the API to `send` and `get` data. However, currently, when a user creates a score the data is created twice instead of once. Please make sure to `fix` this issue as it will help create a better user experience. See below for more details: 
<img width="436" alt="Screenshot 2023-07-27 at 19 51 07" src="https://github.com/BenMKT/Leaderboard/assets/107192076/44ce0eb9-cdd0-4653-a1f1-d51af2df60ba">

Reply:
- Modify the `getScore` function to resolve duplicating scores bug.